apps/stats/management/commands/__init__.py

```diff
@@ -1,12 +1,21 @@
+import json
 import os
 from datetime import datetime, timedelta
 from optparse import make_option
 
 from django.conf import settings
+from django.core.files.base import ContentFile
+from django.core.files.storage import get_storage_class
 from django.core.management.base import BaseCommand
+from django.forms.models import model_to_dict
 
 from .hive_connection import query_to_file
 
+
+storage_class = get_storage_class()
```

Comment:
I had that at first and found it didn't look nice, so changed it, but I'm happy to change it back ;)